### PR TITLE
possibility to open lcp publications

### PR DESCRIPTION
**Pull request by Scarsniik**

### CODE REVIEW

File: src/main/converter/opds.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ export class OpdsFeedViewConverter {
         const title = convertMultiLangStringToString(metadata.Title);
         const authors = convertContributorArrayToStringArray(metadata.Author);
         const publishers = convertContributorArrayToStringArray(metadata.Publisher);
-        const tags = metadata.Subject.map((subject) => subject.Name);
+        const tags = metadata.Subject.map((subject) => subject.Name as string);
```
Comment (danielweck) [inline src/main/converter/opds.ts]: This forced type coercion to `string` is incorrect. The correct implementation is in this older Pull Request:
https://github.com/readium/readium-desktop/pull/202

File: src/main/streamer.ts [hunk 2]
```diff
@@ -122,6 +122,8 @@ function computeReadiumCssJsonMessage(publication: Publication, link: Link | und
         typeScale: readiumCSSDefaults.typeScale,
 
         wordSpacing: settings.wordSpacing,
+
+        reduceMotion: false,
```
Comment (danielweck) [inline src/main/streamer.ts]: I suggest to use `readiumCSSDefaults.reduceMotion`, as in the original Pull Request: https://github.com/readium/readium-desktop/pull/202

File: src/main/converter/opds.ts (hunk 1 above)
Comment (danielweck) [reply src/main/converter/opds.ts]: Fixed in commit: https://github.com/readium/readium-desktop/pull/212/commits/5ec111bf7f326759c1e39ac643da82c4786943a7

File: src/main/streamer.ts (hunk 2 above)
Comment (danielweck) [reply src/main/streamer.ts]: Fixed in this commit: https://github.com/readium/readium-desktop/pull/212/commits/74cbb6be8232b8d41e360efd72d2c4cec05a2439

File: src/common/redux/actions/lcp.ts [hunk 3]
```diff
@@ -31,7 +30,7 @@ export enum ActionType {
     StatusUpdateError = "LCP_STATUS_UPDATE_ERROR",
 }
 
-export function checkUserKey(publication: Publication, hint: string): Action {
+export function checkUserKey(publication: PublicationDocument, hint: string): Action {
```
Comment (clebeaupin) [inline src/common/redux/actions/lcp.ts]: PublicationDocument does not exist in common.
Use PublicationView

**Pull request merged**